q2_sample_classifier/tests/test_classifier.py

```diff
@@ -1147,8 +1147,8 @@ def test_metatable(self):
 
     def test_metatable_missing_error(self):
         with self.assertRaisesRegex(ValueError, "missing values"):
-            sample_classifier.actions.metatable(
-                self.md2, missing_values='error')
+            print(sample_classifier.actions.metatable(
```

Comment:
Drive by comment

```suggestion
            (sample_classifier.actions.metatable(
```

Reply:
ugh you caught my ugly debugging detritus 😞